### PR TITLE
feat: #2127 Admin Portal should be refactored to use new Connect Session Auth module

### PR DESCRIPTION
**Pull request by trankhacvy**
# Pull request checklist

## Does this close any currently open issues?

fixes: https://github.com/reapit/foundations/issues/2127

**Please check if your PR fulfills the following requirements:**

- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [x] Build (`yarn build`) was run locally and any changes were pushed
- [x] Lint (`yarn lint`) has passed locally and any fixes were made for failures
- [x] Test (`yarn test`) has passed locally and any fixes were made for failures

## Pull request type

**Please check the type of change your PR introduces:**

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

Issue Number: #

## What is the new behavior?

-

## Does this introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: packages/admin-portal/src/types/core.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { DevsManagementState } from './../reducers/devs-management'
 import Routes from '../constants/routes'
 import ActionTypes from '../constants/action-types'
-import { AuthState } from '../reducers/auth'
+// import { AuthState } from '../reducers/auth'
```
Comment (tanphamhaiduong) [inline packages/admin-portal/src/types/core.ts:4]: Is this temporary comment ?

File: packages/admin-portal/src/core/connect-session.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { ReapitConnectBrowserSession } from '@reapit/connect-session'
+// Referencing config direct rather than via window because window.reapit.config not available at compile time
+// and throws a runtime error
+import config from '../../config.json'
```
Comment (willmcvay) [inline packages/admin-portal/src/core/connect-session.ts:4]: We can use window.reapit.config now - I worked this out. You just need to refactor the run function in the index.tsx to lazy load the App when the config has been set - see https://github.com/reapit/foundations/blob/master/packages/aml-checklist/src/core/index.tsx#L61

File: packages/admin-portal/src/core/index.tsx [hunk 3]
```diff
@@ -5,10 +5,6 @@ import { render } from 'react-dom'
 import ReactGA from 'react-ga'
```
Comment (willmcvay) [inline packages/admin-portal/src/core/index.tsx:5]: See above, a small refactor of the run function required https://github.com/reapit/foundations/blob/master/packages/aml-checklist/src/core/index.tsx#L61

File: packages/admin-portal/src/core/private-route-wrapper.tsx [hunk 4]
```diff
@@ -20,51 +16,31 @@ export const PrivateRouteWrapper: React.FunctionComponent<PrivateRouteWrapperPro
...
-                <Loader />
-              </Section>
-            }
-          >
-            {children}
-          </Suspense>
-        </FlexContainerResponsive>
-      </FlexContainerBasic>
-    </AppNavContainer>
+    <ReapitConnectContext.Provider value={{ ...session }}>
```
Comment (willmcvay) [inline packages/admin-portal/src/core/private-route-wrapper.tsx:26]: I think Duong has removed the need for the context provider now

**Pull request merged**